test/modules/TestResultCoordinator/Topology.cs

```diff
@@ -1,13 +1,13 @@
-// Copyright (c) Microsoft. All rights reserved.
-namespace TestResultCoordinator
-{
-    public enum Topology
-    {
-        // A single edge device.
-        SingleNode,
-
-        // Multiple edge devices connected together in some toplogy.
-        // Runs an extended test suite relative to `SingleNode`.
-        Nested
-    }
-}
+// Copyright (c) Microsoft. All rights reserved.

```

Comment:
revert?

Reply:
Can you please address this?

Reply:
All I did here was remove the extra newline at the bottom of the file initially

Reply:
I think when your editor saved it might have added some line endings or maybe something else. You can use `git checkout <upstream_branch> -- test/modules/TestResultCoordinator/Topology.cs` to revert.